### ISSUE TITLE
Streams do not update on 1.0.1

### ISSUE THREAD

**Issue by bjacobel** [jasperapp/jasper#182]
After upgrading to version 1.0.1, streams are failing to update when using a GitHub Enterprise instance.

It is likely related to this error:

```
/Applications/Jasper.app/Contents/Resources/app.asar/src/Renderer/Repository/Polling/StreamClient/StreamClient.js:75 TypeError: Cannot read property 'remaining' of null
    at GitHubV4IssueClient.waitRateLimit (/Applications/Jasper.app/Contents/Resources/app.asar/src/Renderer/Library/GitHub/V4/GitHubV4Client.js:54)
    at GitHubV4IssueClient.request (/Applications/Jasper.app/Contents/Resources/app.asar/src/Renderer/Library/GitHub/V4/GitHubV4Client.js:42)
    at async GitHubV4IssueClient.getIssuesByNodeIdsInternal (/Applications/Jasper.app/Contents/Resources/app.asar/src/Renderer/Library/GitHub/V4/GitHubV4IssueClient.js:157)
    at async Promise.all (index 0)
    at async GitHubV4IssueClient.getIssuesByNodeIds (/Applications/Jasper.app/Contents/Resources/app.asar/src/Renderer/Library/GitHub/V4/GitHubV4IssueClient.js:147)
    at async StreamClient.injectV4Properties (/Applications/Jasper.app/Contents/Resources/app.asar/src/Renderer/Repository/Polling/StreamClient/StreamClient.js:183)
    at async StreamClient.search (/Applications/Jasper.app/Contents/Resources/app.asar/src/Renderer/Repository/Polling/StreamClient/StreamClient.js:128)
    at async StreamClient.exec (/Applications/Jasper.app/Contents/Resources/app.asar/src/Renderer/Repository/Polling/StreamClient/StreamClient.js:73)
    at async _StreamPolling.run (/Applications/Jasper.app/Contents/Resources/app.asar/src/Renderer/Repository/Polling/StreamPolling.js:118)
```

Which is thrown from this method:

https://github.com/jasperapp/jasper/blob/6116c671bd4a6987e95d0d268d21e83495be357e/src/Renderer/Library/GitHub/V4/GitHubV4Client.ts#L63-L71

because the response from GitHub Enterprise for this request contains a `rateLimit` field, but the value is null:

```json
{
    "data": {
        "nodes": [
            {
                "node_id": "MDU6SXNzdWU5NjI5MDU=",
                "__typename": "Issue"
                [ ... redacted ... ]
            },
            {
                "node_id": "MDU6SXNzdWU5NjI4Mzg=",
                "__typename": "Issue"
                [ ... redacted ... ]
            }
        ],
        "rateLimit": null
    }
}
```

I see this error every ten seconds when the streams try to refresh.

I am running GitHub Enterprise Server 2.20.16.

**Comment by bjacobel** [comment]
I've opened a PR to potentially fix this issue: https://github.com/jasperapp/jasper/pull/183

**Comment by bjacobel** [comment]
Looks like this is resolved by 5ca0405135a0b24798d91480d285487a3da5f5d7.

**Comment by h13i32maru** [comment]
I fixed this by https://github.com/jasperapp/jasper/commit/5ca0405135a0b24798d91480d285487a3da5f5d7

Thanks.

**Comment by h13i32maru** [comment]
Released v1.0.2
https://jasperapp.io/release.html
